### PR TITLE
Change the app name and all occurences from Study-app to GridStudy

### PR DESCRIPTION
**Pull request by mikatheo**
Signed-off-by: Mika Theodore <michael.theodore@rte-france.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,65 +11,65 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- Geographical display of the imported networks
-- On-demand display of electrical substations
+-   This new CHANGELOG file can be used as a communication support for the users concerning our main exposed functionalities
+-   Manage authentication from an Azure account
+-   Create and visualize studies from UCT, IIDM and CGMES network files
+-   Geographical display of the imported networks
+-   On-demand display of electrical substations
 
 [unreleased]: https://github.com/gridsuite/study-app/compare/v0.4.0...HEAD
 [0.4.0]: https://github.com/gridsuite/study-app/releases/tag/v0.4.0
```
Comment (ChamseddineBhd) [inline CHANGELOG.md:72]: https://github.com/gridsuite/gridstudy/releases/tag/v0.4.0 ?
...

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,65 +11,65 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 [0.3.0]: https://github.com/gridsuite/study-app/releases/tag/v0.3.0
```
Comment (ChamseddineBhd) [inline CHANGELOG.md:73]: /gridstudy/...

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,65 +11,65 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 [0.2.0]: https://github.com/gridsuite/study-app/releases/tag/v0.2.0
```
Comment (ChamseddineBhd) [inline CHANGELOG.md:74]: /gridstudy/...

File: CHANGELOG.md [hunk 4]
```diff
@@ -11,65 +11,65 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-[0.1.0]: https://github.com/gridsuite/study-app/releases/tag/v0.1.0
\ No newline at end of file
+[0.1.0]: https://github.com/gridsuite/study-app/releases/tag/v0.1.0
```
Comment (ChamseddineBhd) [inline CHANGELOG.md:75]: /gridstudy/...

File: package.json [hunk 5]
```diff
@@ -1,10 +1,10 @@
 {
-  "name": "study-app",
+  "name": "gridstudy",
   "version": "0.1.0",
   "homepage": ".",
   "private": true,
   "dependencies": {
-    "@gridsuite/commons-ui": "^0.1.1",
+    "@gridsuite/commons-ui": "file:../commons-ui",
```
Comment (ChamseddineBhd) [inline package.json:7]: reset that to @gridsuite/commons-ui:^0.2.0

File: CHANGELOG.md (hunk 1 above)
Comment (mikatheo) [reply CHANGELOG.md:72]: I had a rebase problem, I'm fixing it :+1:

File: .github/workflows/build.yml [hunk 6]
```diff
@@ -33,4 +33,4 @@ jobs:
               uses: gridsuite/broadcast-event@master
               with:
                   token: ${{ secrets.REPO_ACCESS_TOKEN }}
-                  event-type: study_app_updated
+                  event-type: gridstudy_updated
```
Comment (jonenst) [inline .github/workflows/build.yml:36]: I prefer gridstudy_app_updated

**Review by jonenst — APPROVED** [review]

**Review by jonenst — APPROVED** [review]

**Pull request merged**